src/test/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/CCJRequestSubmittedNotificationEmailContentProviderTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff.content;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailTemplates;
+import uk.gov.hmcts.cmc.claimstore.services.TemplateService;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CCJRequestSubmittedNotificationEmailContentProviderTest {
+
+    private static final String EMAIL_SUBJECT = "Email Subject";
+    private static final String EMAIL_BODY = "Email Body";
+    private CCJRequestSubmittedNotificationEmailContentProvider provider;
+    @Captor
+    private ArgumentCaptor<Map<String, Object>> mapCaptor;
+
+    @Mock
+    private TemplateService templateService;
+
+    @Mock
+    private StaffEmailTemplates staffEmailTemplates;
+
+    @Before
+    public void setup() {
+        provider = new CCJRequestSubmittedNotificationEmailContentProvider(templateService, staffEmailTemplates);
+
+        when(staffEmailTemplates.getCCJRequestSubmittedEmailSubject()).thenReturn(EMAIL_SUBJECT);
+        when(staffEmailTemplates.getCCJRequestSubmittedEmailBody()).thenReturn(EMAIL_BODY);
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void shouldThrowNullPointer() {
+        provider.createContent(null);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void shouldThrowIllegalArgument() {
+        provider.createContent(Collections.emptyMap());
+    }
+
+    @Test
+    public void shouldCreateValidContent() {
```

Comment:
this test looks stupid

Reply:
??

Reply:
verifying the number of times things are called is implementation detail:
```
       verify(staffEmailTemplates, times(1)).getCCJRequestSubmittedEmailSubject();
         verify(staffEmailTemplates, times(1)).getCCJRequestSubmittedEmailBody();
         verify(templateService, times(2)).evaluate(any(), mapCaptor.capture());
```